packages/session-recorder/src/BatchLogProcessor.ts

```diff
@@ -26,47 +26,142 @@ export interface BatchLogProcessorConfig {
 }
 
 export class BatchLogProcessor {
```

Comment:
Ho does it differ from OTel BatchLogProcessor?


https://github.com/open-telemetry/opentelemetry-js/blob/main/experimental/packages/sdk-logs/src/platform/browser/export/BatchLogRecordProcessor.ts#L21

Reply:
It's basically the same, but uses Splunk custom `Log` type instead of `ReadableLogRecord` from OTel.

Reply:
As we were discussing recently, by using `LogRecord` from OTel would simplify things a lot.